azure-plugin/generator/Azure.Generator/src/AzurePlugin.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp;
+using Microsoft.Generator.CSharp.ClientModel;
+using System.ComponentModel.Composition;
+
+namespace Azure.Generator;
+
+/// <summary>
+/// XXX
+/// </summary>
+[Export(typeof(CodeModelPlugin))]
+[ExportMetadata("PluginName", nameof(AzurePlugin))]
+public class AzurePlugin : ClientModelPlugin
```

Comment:
Have we closed on the name for this?  Should it be AzureClientPlugin?

Reply:
+1 to Anne's suggested name.

Reply:
The name is not settled, as well as the folder structure.
I just wanted to get started with the discussion with this initial PR.

Reply:
renamed to `AzureClientPlugin`

Reply:
For context this library won't be shipped anywhere, not saying this isn't a better name but it would be public.